apps/avifenc.c

```diff
@@ -2276,6 +2295,7 @@ int main(int argc, char * argv[])
                                     /*ignoreExif=*/AVIF_TRUE,
                                     /*ignoreXMP=*/AVIF_TRUE,
                                     /*allowChangingCicp=*/AVIF_FALSE,
+                                    settings.ignoreGainMap,
```

Comment:
`ignoreGainMap` like above?

Reply:
This is for the grid case, so I don't think we need to be concerned about animations or progressive images. These are all mutually exclusive, right?

Reply:
> These are all mutually exclusive, right?

Currently, yes.